### PR TITLE
Fixes#379: forgot password is now working

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #379 

Changes: Forgot password function added as props in login dialog.

Demo Link:http://susimadhav.surge.sh/
@rishiraj824 @uday96 @isuruAb Please review.
Thanks

### CODE REVIEW

File: src/components/ChatApp/MessageSection/DialogSection.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ export default class DialogSection extends Component {
           bodyStyle={this.props.bodyStyle}
           contentStyle={{width: '35%',minWidth: '300px'}}
           onRequestClose={this.props.onRequestClose()}>
-          <Login {...this.props} />
+          <Login {...this.props}
```
Comment (rishiraj824) [inline src/components/ChatApp/MessageSection/DialogSection.js]: Aren't these changes already handled by https://github.com/fossasia/chat.susi.ai/pull/377/files ?

Comment (madhavrathi) [inline src/components/ChatApp/MessageSection/DialogSection.js]: Coincidently they do. Please tell how to proceed.
Thanks

Comment (rishiraj824) [inline src/components/ChatApp/MessageSection/DialogSection.js]: It's going to be a conflict for either of PRs @madhavrathi @uday96

**Comment by madhavrathi** [comment]
@uday96 Closing this PR as this was solved by https://github.com/fossasia/chat.susi.ai/pull/377/

**Pull request closed without merging**